test/ClientTest.cpp

```diff
@@ -26,6 +26,7 @@
 
 using namespace Casper;
 
+std::string privKeyPemFile = "test/data/matterficasper1_secret_key.pem";
```

Comment:
Shouldn't we add this `.pem` file to the repository?

Reply:
Nope, I asked Jakub

Reply:
hm, that could be a problem, because we need this file to have working tests, we have it in our rpc but it's private repo, generally you can add right now info to readme about it :)